app/models/api/task.rb

```diff
@@ -1,5 +1,8 @@
 module API
   class Task < Base
     has_one :framework
+
+    # GET /tasks/:id/complete
```

Comment:
Small typo - comment says `GET`, but endpoint is defined as a `POST`

Reply:
Good spot! 👍